### PR TITLE
Migration and seed for groups

### PR DESCRIPTION
**Pull request by Pallabi-ghosh**
# Description

I have created the database table and I did it with Knex. Also, add a seed file for development to add a couple of initial groups.
I had a problem to understand this first but now its clear. 

Fixes # (22)

# How to test?

This time there was no error for commit and push my branch. 

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by nuha90** [comment]
do you need date in your table pallabi ?

File: src/server/migrations/20201117134811_create_groups_schema.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('group_table', (table) => {
```
Comment (tpetrina) [inline src/server/migrations/20201117134811_create_groups_schema.js:2]: We don’t put `table` in table names. `groups` is enough,

File: src/server/migrations/20201117134811_create_groups_schema.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    table.increments();
+    table.string('title').notNullable();
+    table.datetime('startDate').notNullable();
```
Comment (tpetrina) [inline src/server/migrations/20201117134811_create_groups_schema.js:5]: Do we need groups to have start/end date? What would be the start/end date for group `class 14`?
These are not study groups but groups a student belongs to.

**Review by tpetrina — CHANGES_REQUESTED** [review]
Remove client side files.
Also, title of the PR is not clear, it should be “Migration and seed for groups”

**Comment by Pallabi-ghosh** [comment]
I have removed those client side file also change the PR name. Could you please take a look and approve this PR.

**Comment by Pallabi-ghosh** [comment]
Closing the PR because of too many commits.

**Pull request closed without merging**